test/unittests/skills/test_skill_manager.py

```diff
@@ -139,48 +128,6 @@ def test_remove_git_locks(self):
 
         self.assertFalse(path.exists(git_lock_file_path))
 
-    def test_load_priority(self):
```

Comment:
Should these tests be updated/replaced?

Reply:
They would need a full rewrite, dont really make sense without msm...

We need to reenable tests and add a github action, but theres still some broken tests that were never updated. Holmes removed all tests and i just got them back for ovos, but they need a full review. And we need a lot of new tests if someone is up to it

Reply:
The functionality is still there for priority skills load though, right? I guess either way it's out of scope of this PR to do all the test cleanup, so up to you if you think a stub or TODO is needed. I'll open an issue since tests will likely be several PR's to get to a state where we're happy with them

Reply:
Need to look at the tests, but default skills list comes from msm and priority loading is supposed to install them, so these tests still do not really apply... full test rewrite is warranted